test/spec/unit/core/targeting_spec.js

```diff
@@ -392,7 +392,7 @@ describe('targeting tests', function () {
       beforeEach(function() {
         bid4 = utils.deepClone(bid1);
         bid4.adserverTargeting['hb_bidder'] = bid4.bidder = bid4.bidderCode = 'appnexus';
-        bid4.cpm = 0;
+        bid4.cpm = 0.01;
```

Comment:
I think this means that deal bids with 0 cpm used to be included, and now are excluded by default. If it's intentional, I think it should be tested explicitly.

Reply:
The way these tests were setup they were inheriting configs from previously run tests. This test does not explicitly allow $0.00 cpms so I made it a usable bid. It can also be updated to set the config to allow zero dollar cpms but it is not relevant to the test itself

Reply:
Oh hmm maybe I missed something here. Let me take a look. I thought this was inheriting a config by mistake

Reply:
So Im not sure if this was correct behavior. The test is not within the send all deals description nor the send zero cpms. Since it involves deals, I am configuring it to send all deals, and I updated the logic to ignore $0 cpms in that instance. It also worked configuring send zero dollar cpm but deals made more sense based on the test